doc/manual.rst

```diff
@@ -490,11 +490,17 @@ this. Another reason is that Nim can thus support `array[char, int]` or
 type is used for Unicode characters, it can represent any Unicode character.
 `Rune` is declared in the `unicode module <unicode.html>`_.
 
+A character literal that  does not end in ``'`` interpreted as ``'`` if there
```

Comment:
I don't understand what this sentence means.
is that to distinguish:
```
proc `'f'foo`(a: string) = discard # 'f' char lit
vs
proc `'ffoo`(a: string) = discard # no char lit
```
?

either way, it needs clarification

Reply:
"no char lit" is not correct, it's a character literal either way. The single unfinished quote is interpreted as a single quote. That means ``'`` == ``'\''``